### DISCUSSION TITLE
equation reference broken

### DISCUSSION THREAD

**Discussion by yairmau** [quarto-dev/quarto-cli#6349] [Q&A]

**Comment by cderv** [comment]
Hi 

Can you update you description to explain actual question or issue ? 

Thank you

**Reply by mcanouil** [reply]
To note, the post was edited and contained initially
<img width="629" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/4f017fd9-c6e9-47ec-a3d4-0be2b4942685">

Since the description was removed by the author (@yairmau ) unless stated otherwise I will close this.

**Reply by cderv** [reply]
Yeah I saw that already. I just asked to have the real update by @yairmau directly. I did not want to assume this was correct without confirmation. But thanks for putting a screenshot in the thread !

@yairmau if this is still the problem please do edit again your post. Thank you
